README.md

```diff
@@ -1,98 +1,112 @@
+yourself.
 
-## About 
-Adds the GOVUK Frontend for Rails using the asset pipeline.
+## About
+
+Adds the GOV.UK Frontend for Rails using the asset pipeline.
 
 https://github.com/alphagov/govuk-frontend
 
 ## Breaking Changes in v3.0.0
 
-This release of the GOVUK Frontend has breaking changes! Do not update to this
+This release of the GOV.UK Frontend has breaking changes! Do not update to this
 version until you have read and understood:
 
-You will not be required to updated any file paths with govuk as we take care of
-that for you, all others changes will need to be managed:
+You will not be required to updated any file paths with `govuk` as we take care
+of that for you, all others changes will need to be managed:
 
 https://github.com/alphagov/govuk-frontend/releases/tag/v3.0.0
 
 ## Limitations
 
-This gem does not include the ie8 version of GOVUK Frontend, as the service
-standard no longer requires it:
+This gem does not include the `ie8` version of GOV.UK Frontend, as the service
+standard no longer requires support for Internet Explorer versions less than 11:
 
 https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices
 
-This gem and its documentation cannot tell you how to use the GOVUK effectively,
-see the GOVUK Design System documentation for that:
+This gem and its documentation cannot tell you how to use the GOV.UK patterns
+effectively, see the GOV.UK Design System documentation for that:
 
-https://design-system.service.gov.uk/ 
+https://design-system.service.gov.uk/
 
 ## Installation
 
 Add this line to your application's Gemfile:
 
-`ruby gem 'dxw_govuk_frontend_rails'`
+```ruby
+gem 'dxw_govuk_frontend_rails'
+```
 
 And then execute:
 
-`bundle`
+```
+$ bundle
+```
+
+## Usage
+
+Your Rails app will need to have SCSS enabled.
 
-## Usage Your Rails app will need to have sass enabled.
+**IMPORTANT: You must not use this library without changes if the service you
+are building is not on GOV.UK. Read and understand the guidance here:**
+
+https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk#if-your-service-isnt-on-govuk
 
 ### Basic usage
 
-If the service you are building does not need to modify or extend the GOVUK
-Frontend all you need to do is import the main sass and javascript files into
+If the service you are building does not need to modify or extend the GOV.UK
+Frontend, all you need to do is import the main SCSS and JavaScript files into
 your Rails app:
 
-- create a sass file in your application: 
+- create a SCSS file in your application:
 
-`app/assets/stylesheets/govuk_frontend_rails.scss`
-- import the styles into `govuk_frontend_rails.scss`: 
+  `app/assets/stylesheets/govuk_frontend_rails.scss`
 
-`@import "govuk-frontend-rails";`
-- require the `govuk-frontend-rails.scss` file in `application.css` or
-  equivalent: 
+- import the styles into `govuk_frontend_rails.scss`:
+
+  `@import "govuk-frontend-rails";`
```

Comment:
If we're importing the css from `govuk_frontend_rails` in the next step, I don't think we actually need this step at all. I think doing this will result in including the styles in our bundle twice. We should advocate for one approach or the other, not both.